torch/_dynamo/output_graph.py

```diff
@@ -745,12 +746,9 @@ def get_submodule(self, keys):
         return obj
 
     def new_var(self, name="tmp"):
-        existing = set(self.code_options["co_varnames"])
-        for i in itertools.count():
-            var = f"{name}_{i}"
-            if var not in existing:
-                self.code_options["co_varnames"] += (var,)
-                return var
+        var = f"{name}_{next(self.unique_var_id)}"
```

Comment:
I think we still need the loop because the user code could have a variable name `tmp0` `tmp1` etc.

Though we could just put a loop around this code and in the common case it will be O(1)

Reply:
Good point. Updated.